### PR TITLE
Update navbar

### PR DESCRIPTION
**Pull request by sullyo**
- Add new sidebar closed icon
- Update mobile navbar in milkyway to support props for sidebar in host
- Add overlay and animation for button clicks on sidebar overlays
- Update hamburger icon

Preview video here (since we cant have a tesseract env)
https://user-images.githubusercontent.com/13056592/126521386-6209b4fe-6c00-4120-bab5-4fa75074ba04.mov

### CODE REVIEW

File: components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx [hunk 1]
```diff
@@ -1,22 +1,53 @@
-
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (MichaelZhou) [inline components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx:1]: Importing `useEffect` here but it looks like it isn't being used.

File: components/Navbar/NavbarTabletMobile/NavbarTabletMobile.module.less [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  min-height: 100vh !important;
+}
+.bottom {
+  margin-top: auto;
+}
+
+.menu > a {
+  padding: 16px 24px !important;
+}
+.menu > a:nth-last-child(-n + 5) {
```
Comment (manpreetnarang) [inline components/Navbar/NavbarTabletMobile/NavbarTabletMobile.module.less:26]: `-n + 5` ?

File: components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx [hunk 3]
```diff
@@ -1,22 +1,53 @@
...
+            content={<Icon className={'universe-sidebar-closed'} size="big" />}
+            onClick={() => setSidebarVisible(!sidebarVisible)}
+          ></Button>
+        </Menu.Item>
+        <Menu.Item>
+          <Logo />
+        </Menu.Item>
+        <Menu.Item>
+          <Button
+            className={styles.button + ' ' + styles.largerHamburger}
```
Comment (manpreetnarang) [inline components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx:34]:
```suggestion
            className={`${styles.button} ${styles.largerHamburger}`}
```

Comment (manpreetnarang) [reply components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx:34]: You have to import classNames. Check other components for example.

File: components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx [hunk 4]
```diff
@@ -1,22 +1,53 @@
...
+            content={<Icon className="universe-hamburger-menu" size="big" />}
+            onClick={() => setMenuVisible(!menuVisible)}
+          ></Button>
+        </Menu.Item>
+      </Menu>
+      <MenuTabletMobile visible={menuVisible} user={user} />
+      <div className={sidebarVisible ? styles.sidebar + ' ' + styles.slideInSidebar : styles.sidebar}>{children}</div>
```
Comment (manpreetnarang) [inline components/Navbar/NavbarTabletMobile/NavbarTabletMobile.jsx:41]: same as above. Use classNames.

File: components/Navbar/NavbarTabletMobile/MenuTabletMobile.jsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      position="right"
+      className={visible ? styles.menu + ' ' + styles.slideInRight : styles.menu}
+    >
+      <UniverseHome />
+      <Menu.Item href="/explore">Find Events</Menu.Item>
+      <Menu.Item href="/create">Create Event</Menu.Item>
+      <Menu.Item href="/features">Features</Menu.Item>
+      <Menu.Item href="/customers">Customers</Menu.Item>
+      <Menu.Item href="/pricing">Pricing</Menu.Item>
+      <Menu.Item className={styles.menuBorder + ' ' + styles.bottom} href="/help">
```
Comment (manpreetnarang) [inline components/Navbar/NavbarTabletMobile/MenuTabletMobile.jsx:64]: use `classNames`

File: components/Navbar/NavbarTabletMobile/MenuTabletMobile.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+      </Menu.Item>
+      <Menu.Item href="/sign-in?redirectTo=https://www.universe.com/">Login</Menu.Item>
+      <Menu.Item href="/sign-up">Sign Up</Menu.Item>
+    </Menu>
+  );
+};
+const UniverseHome = () => (
+  <Menu.Item href="/">
+    <Logo className={styles.universeLogo} />
+  </Menu.Item>
```
Comment (manpreetnarang) [inline components/Navbar/NavbarTabletMobile/MenuTabletMobile.jsx:75]: I believe we need internationalization here.

**Review by manpreetnarang — CHANGES_REQUESTED** [review]
internationalization and use `classNames`

File: components/Navbar/NavbarTabletMobile/NavbarTabletMobile.module.less (hunk 2 above)
Comment (sullyo) [reply components/Navbar/NavbarTabletMobile/NavbarTabletMobile.module.less:26]: The last 5 menu items

**Comment by sullyo** [comment]
> internationalization and use `classNames`

Updated :)

File: components/Navbar/NavbarComputer/NavbarComputer.jsx [hunk 7]
```diff
@@ -94,4 +93,8 @@ const Account = ({ user }) => {
   );
 };
 
-const UniverseHome = () => <Menu.Item href="/"><Logo /></Menu.Item>;
+const UniverseHome = () => (
+  <Menu.Item href="/">
+    <Logo />
+  </Menu.Item>
+);
```
Comment (makeready) [inline components/Navbar/NavbarComputer/NavbarComputer.jsx:100]: This file is missing a bunch of i18n as well. The non-internationalized strings here weren't added in this PR, so it's your choice whether to consider those fixes as being in scope for your ticket.

Comment (sullyo) [reply components/Navbar/NavbarComputer/NavbarComputer.jsx:100]: At this time we aren't using this component for desktop. When we decide to switch to the milkyway navbar for desktop i'll make those changes.

**Review by makeready — APPROVED** [review]

**Pull request merged**